frontend/src/app.d.ts

```diff
@@ -10,6 +10,14 @@ declare global {
     lexbox: { fetchProxy: Fetch }
   }
 
+  type LexboxResponseHandlingConfig = {
+    disabledStatusCodes?: number[];
```

Comment:
I'd rather be more explicit about what we are using this for. To use this property correctly you need to know where it's consumed and what it's actually doing.
I'd rather something like `disableRedirectOnAuthError` or `ignoreAuthErrors` which is just a boolean property.

Reply:
That works for me.